gen_config.py

```diff
@@ -0,0 +1,161 @@
+import typing as t
```

Comment:
this will either be added to neuro/apolo-extras, separate package or added to app deployment controller, wdyt?

Reply:
do you see this as something that will be reused frequently?

Reply:
Not really, but the context from here, at least, outputs after the deployment, could be used for other apps.

Later, this will lay on the Apps API